bigquery/indexer.py

```diff
@@ -54,6 +54,26 @@ def open_and_return_json(file_path):
     return jsonDict
 
 
+# Keep in sync with convert_to_index_name() in data-explorer repo.
+def convert_to_index_name(s):
```

Comment:
Suggestion:
```
prohibited_chars = [' ', '"', '*',...]
for char in prohibited_chars:
    s = s.replace('char', '_');
```

That would probably eliminate the need for a comment about what this code is doing

Reply:
Change code. Which comment did you want to delete? Both seem useful..